app/services/push_notifications/send.rb

```diff
@@ -0,0 +1,22 @@
+module PushNotifications
+  class Send
+    def self.call(user = nil, **kwargs)
```

Comment:
We seem to have only 3 kwargs and all of them seem necessary since there are no default values. Maybe make them explicit?

Reply:
✅ Switched this signature to required keyword arguments